pkgs/servers/gotosocial/default.nix

```diff
@@ -43,8 +43,17 @@ buildGoModule rec {
   # tests are working only on x86_64-linux
   doCheck = stdenv.isLinux && stdenv.isx86_64;
 
-  # flaky test
-  checkFlags = [ "-skip=^TestPage/minID,_maxID_and_limit_set$" ];
+  checkFlags =
+    let
+      # flaky / broken tests
+      skippedTests = [
+        "TestPage/minID,_maxID_and_limit_set"
```

Comment:
Could you please also add this link as a comment? I've forgot to add it.
https://github.com/superseriousbusiness/gotosocial/issues/2651

Reply:
Done! Updated.